src/ir_York.cpp

```diff
@@ -0,0 +1,358 @@
+// Copyright 2022 Daniele Gobbetti
+/// @file
+/// @brief Support for the York AC protocol (AC model: MHH07P17 remote GRYLH2A).
+
+// Supports:
+//   Brand: York,  Model: MHH07P17
+// Note: Most of the code is autogenerated by the provided tools or assembled
+// from other support classes
+
+#include "ir_York.h"
+#include <algorithm>
+#include <cstring>
+#ifndef ARDUINO
+#include <string>
+#endif
+#include "IRrecv.h"
+#include "IRremoteESP8266.h"
+#include "IRsend.h"
+#ifdef UNIT_TEST
+#include "IRsend_test.h"
+#endif
+#include "IRtext.h"
+#include "IRutils.h"
+
+
+
+
+using irutils::addBoolToString;
+using irutils::addModeToString;
+using irutils::addFanToString;
+using irutils::addTempToString;
+using irutils::addSwingVToString;
+using irutils::addLabeledString;
+using irutils::minsToString;
+
+
+// Constants
+const uint16_t kHdrMark = 4887;
+const uint16_t kBitMark = 612;
+const uint16_t kHdrSpace = 2267;
+const uint16_t kOneSpace = 1778;
+const uint16_t kZeroSpace = 579;
+const uint16_t kFreq = 38000;  // Hz. (Guessing the most common frequency.)
+const uint16_t kOverhead = 3;
+
+#if SEND_YORK
+// Alternative >64bit function to send YORK messages
+// Function should be safe over 64 bits.
```

Comment:
Remove these two lines. They are just a note to the programmer from the automatically generated code.